code/network/multiutil.cpp

```diff
@@ -842,7 +843,7 @@ int multi_create_player( int net_player_num, player *pl, const char* name, net_a
 void multi_make_player_ai( object *pobj )
 {
 
-	Assert ( pobj != NULL );
+	Assert(pobj != NULL);
```

Comment:
you missed a nullptr here

Reply:
I think I have this replaced with an assertion but I forgot to commit the changes.